src/parser/formatting-evaluator-module.ts

```diff
@@ -139,17 +141,36 @@ export class FormattingEvaluatorModule implements Module {
 
     for (const element of elements) {
       const tagName = element.tagName.toLowerCase();
-      const wordCount = this._countWords(this._multipliers[commentType].regex, element.textContent || "");
+      // We cannot use textContent otherwise we would duplicate counts, so instead we extract text nodes
```

Comment:
textContent of the top level parent element will do the right thing.

Reply:
Correct, will be part of https://github.com/ubiquibot/conversation-rewards/issues/92